libs/template/src/lib/push/push.pipe.ts

```diff
@@ -84,13 +84,13 @@ export class PushPipe<U> implements PipeTransform, OnDestroy {
     const scope = (cdRef as any).context;
     this.subscription = this.templateObserver.values$
       .pipe(
-        filter<RxNotification<U>>(
+        filter<RxNotification<any>>(
           (n) =>
             n.kind === RxNotificationKind.suspense ||
             n.kind === RxNotificationKind.next
         ),
-        tap<RxNotification<U>>((notification) => {
-          this.renderedValue = notification.value as U;
+        tap<RxNotification<any>>((notification) => {
+          this.renderedValue = notification.value as any;
```

Comment:
same goes for this one:
```suggestion
        tap(notification => {
          this.renderedValue = notification.value;
```

Reply:
Agreed.